themes/default/css/_besocial/index_besocial.css

```diff
@@ -1663,19 +1663,19 @@ a.help .icon {
 }
 
 .i-comment::before {
-	content: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' stroke='black' stroke-width='6em' fill='%231ab31a' viewBox='0 0 3584 3584'%3E%3Cpath d='M3584 1792q0 348-240 643t-652 466-900 171q-140 0-290-16-396 350-920 484-98 28-228 44-34 4-61-18t-35-58v-2q-6-8-1-24t4-20 9-19l12-18t14-17 16-18q14-16 62-69t69-76 62-79 65-102 54-118 52-152q-314-178-495-440t-181-562q0-260 142-497t382-409 572-273 696-101q488 0 900 171t652 466 240 643z'/%3E%3C/svg%3E");
+	content: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' stroke='black' stroke-width='6em' fill='#43a8da' viewBox='0 0 3584 3584'%3E%3Cpath d='M3584 1792q0 348-240 643t-652 466-900 171q-140 0-290-16-396 350-920 484-98 28-228 44-34 4-61-18t-35-58v-2q-6-8-1-24t4-20 9-19l12-18t14-17 16-18q14-16 62-69t69-76 62-79 65-102 54-118 52-152q-314-178-495-440t-181-562q0-260 142-497t382-409 572-273 696-101q488 0 900 171t652 466 240 643z'/%3E%3C/svg%3E");
```

Comment:
Do unencoded hashmarks work in all browsers here? Namely IE9?

Would simplify things, it has gotten annoying.